ui/src/app/components/help-sidebar.tsx

```diff
@@ -831,7 +844,7 @@ export const HelpSidebar = fp.flow(
                 <TooltipTrigger content={<div>{icon.tooltip}</div>} side='left'>
                   <div style={activeIcon === icon.id ? iconStyles.active : icon.disabled ? iconStyles.disabled : styles.icon}
                        onClick={() => {
-                         if (icon.id !== 'dataDictionary' && !icon.disabled) {
+                         if (icon.id !== 'dataDictionary' && icon.id !== 'terminal' && !icon.disabled) {
```

Comment:
Lets move this into the icon config; something like `canToggle` or `canActivate` or `hasContent`, which is true for the other icons configs, false for terminal and dataDictionary (since they cannot be "toggled on", they're just buttons)